frontend/app/src/locales/en.json

```diff
@@ -609,6 +612,21 @@
     "stability_fee": "Stability fee",
     "total_lost": "Total lost due to interest"
   },
+  "manage_custom_assets": {
+    "backup": {
+      "button": "Backup",
+      "subtitle": "Create a backup of your custom assets",
+      "success": "The backup was successful",
+      "title": "Backup"
+    },
+    "restore": {
+      "button": "Restore",
+      "subtitle": "Restore custom assets from a backup",
```

Comment:
Maybe we should say that you can upload the exported zip?

Reply:
Sure I will change it when I update the text from backup/restore to import/export